misc/theme-cicada/src/opnsense/www/themes/cicada/assets/stylesheets/bootstrap-select/less/bootstrap-select.less

```diff
@@ -0,0 +1,332 @@
+/*!* {
+    Bootstrap-select {
+        v1.9.3 {
+            (http://silviomoreto.github.io/bootstrap-select)** {
+                Copyright {
+                    2013-2015 {
+                        bootstrap-select* {
+                            Licensed {
+                                under {
+                                    MIT {
+                                        (https://github.com/silviomoreto/bootstrap-select/blob/master/LICENSE)*/span.form-control {
```

Comment:
format

Reply:
Thanks @fabianfrz  - I'll correct these this morning.

Reply:
No idea... as I said bellow.

Done apart from the format issue for bootstrap-select.less. No matter which way I try and tidy that it still looks wrong.

Reply:
let me take a look post merge, thanks!